client/blocks/upload-image/index.jsx

```diff
@@ -231,10 +234,12 @@ class UploadImage extends Component {
 
 		const isEditingDefaultImage = defaultImage && selectedImage === defaultImage.URL;
 
-		const media = isEditingDefaultImage ? defaultImage : {
-			src: selectedImage,
-			file: selectedImageName
-		};
+		const media = isEditingDefaultImage
+			? defaultImage
+			: {
```

Comment:
That's a known issue where the Prettier formatting is incompatible with the opinion of the eslint `indent` rule. I recommend keeping the Prettier formatting, commit with the `-n` flag and also ignore the complaints from the CircleCI lint and test runs (just make sure that this is really the only error!)

The formatting-only eslint rules are going to be disabled soon for files with the `@format` marker, once we find a good way to do it :)